src/ToastNotification/res/index.js

```diff
@@ -42,3 +44,52 @@ on(
     _closeToastNotification(e.delegateTarget);
   }
 );
+
+export function addNotification(container, title, content, removable, color)
+{
+  const toastContainer = document.createElement('div');
+  toastContainer.classList.add('toast-notification');
+
+  if(removable)
+  {
+    toastContainer.classList.add('toast-notification--removable');
+  }
+
+  if(!color)
+  {
+    color = 'blue';
+  }
+
+  toastContainer.classList.add('toast-notification--with-color');
+  toastContainer.classList.add(`bgc-${color}`);
+  toastContainer.classList.add(`brdrc-${color}`);
+
+  const toastContent = document.createElement('div');
+  toastContent.classList.add('toast-notification__content');
+  toastContainer.appendChild(toastContent);
+
+  if(title)
+  {
+    const toastTitle = document.createElement('div');
+    toastTitle.classList.add('toast-notification__title');
+    toastTitle.innerHTML += title;
+
+    toastContent.appendChild(toastTitle);
+  }
+
+  if(content)
+  {
+    const toastDescription = document.createElement('div');
+    toastDescription.classList.add('toast-notification__description');
+    toastDescription.innerHTML += content;
```

Comment:
should probably be innerText
```suggestion
    toastDescription.innerText += content;
```